test/integration/connectors/env_setup/qdrant/docker-compose.yaml

```diff
@@ -0,0 +1,5 @@
+services:
```

Comment:
If we're just running one container, you don't need to use the docker compose approach for testing but can instead use the docker library.  Take a look at how that's done in the snowflake test: [test_snowflake.py#L84](https://github.com/Unstructured-IO/unstructured-ingest/blob/main/test/integration/connectors/sql/test_snowflake.py#L84)